www/wml/Portfile

```diff
@@ -63,12 +64,11 @@ configure.args          --with-perl="${perl5.bin}" \
 
 configure.env           CPATH=${prefix}/include LANG=C LC_ALL=C
 
-build.args              CC=${configure.cc} CPP=${configure.cpp}
+build.args              CC="${configure.cc} -Wno-error=implicit-function-declaration" CPP=${configure.cpp} 
```

Comment:
instead of disable the error due to implicit function declaration, you should ideally fix the issue